test/unittests/mongoBackend/mongoContextProvidersUpdate_test.cpp

```diff
@@ -523,20 +523,27 @@ TEST(mongoContextProvidersUpdateRequest, noPatternAttrOneSingle)
   EXPECT_EQ(SccOk, ms);
 
   EXPECT_EQ(SccNone, res.errorCode.code);
```

Comment:
You are right. The way in which it works now it's not coherent. If servicePath gets a SccNone then it should return a 500 Internal Error (as that would mean that mongoBackend has some problem doing its work). This should be SccOk.

However, the problem affect to many places:

```
fermin@debvm:~/src/fiware-orion/test/unittests/mongoBackend$ grep -c SccNone *.cpp | grep -v '0$'
mongoContextProvidersQuery_test.cpp:19
mongoContextProvidersUpdate_test.cpp:10
mongoDiscoverContextAvailability_test.cpp:79
mongoQueryContextFilterExistEntity_test.cpp:3
mongoQueryContext_test.cpp:40
mongoSubscribeContextAvailability_test.cpp:24
mongoSubscribeContext_test.cpp:65
mongoUpdateContextAvailabilitySubscription_test.cpp:25
mongoUpdateContextSubscription_test.cpp:64
mongoUpdateContext_withOnchangeSubscriptions_test.cpp:1
```
What I suggest is to fix this in the *_test.cpp touched by this PR, leave the others for future PRs (probably serviceRoutine logic need to be adapted also to make right use of SccOk/SccNone?)